frontend/pages/admin/IntegrationsPage/cards/IdentityProviders/IdentityProviders.tsx

```diff
@@ -1,15 +1,13 @@
 import React from "react";
 
 import IdentityProviderSection from "./components/IdentityProviderSection";
-import EndUserAuthSection from "./components/EndUserAuthSection";
 
 const baseClass = "identity-providers";
 
 const IdentityProviders = () => {
   return (
     <div className={baseClass}>
       <IdentityProviderSection />
```

Comment:
Per discussions with @noahtalerman earlier today, this sticks around as its own tab, but is now limited to IdP/SCIM status.